sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -464,6 +471,18 @@ def __init__(self, server_url, proxy=None,
         self.https = (self.scheme == 'https')
         self.proxy = None
         self.timeout = timeout
+        if isinstance(client_cert, tuple) and len(client_cert) >= 2:
```

Comment:
this conditional is confusing. Recommend checking for len of client_cert once and once only, refactor this to validate that client_cert is tuple and break into a conditional statement or using a ternary operator.

Reply:
I did think about this. Do you have a recommended place to format the client cert? 

My initial thought was during config validation. But I didn’t think it fit well there from an immutability standpoint.